ha/k8s_setup/ha_setup.py

```diff
@@ -177,6 +195,7 @@ def process(self):
             # Dummy value fetched for now. This will be replaced by the key/path for the pod label onces that is avilable in confstore
             # Ref ticket EOS-25694
             data_pod_label = Conf.get(self._index, f'cortx{_DELIM}common{_DELIM}product_release')
+            # cluster_id = Conf.get(self._index, f'node{_DELIM}{machine_id}{_DELIM}cluster_id')
```

Comment:
If we are getting cluster Id here remove the code for the same from process function, otherwise remove this (commented)  line

Reply:
Removed it from here.